libnmstate/nm/port_vlan.py

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+
+class PortVlanFilter(object):
+    def __init__(self, trunk_tags, access_tag, enable_native_vlan):
+        self._trunk_tags = trunk_tags
+        self._access_tag = access_tag
+        self._is_native_vlan = enable_native_vlan
+
+    def generate(self):
```

Comment:
Won't it fit the `to_*` pattern you used below? e.g. `to_nm`

Reply:
Done